folly/test/JsonTest.cpp

```diff
@@ -22,6 +22,8 @@
 
 #include <folly/portability/GTest.h>
 
+FOLLY_GNU_DISABLE_WARNING("-Wmissing-field-initializers")
```

Comment:
Where are these showing up? We should generally be clear of this particular warning.